### PR TITLE
Update code on mvvm samples for SimpleCalc

### PR DESCRIPTION
**Pull request by pictos**
I updated the `MainViewModel.IsDark` code to use correctly the features of MVVM Toolkit.

### CODE REVIEW

File: reference/SimpleCalc/MVVM-CSharp/SimpleCalculator/MainViewModel.cs [hunk 1]
```diff
@@ -1,24 +1,14 @@
-﻿using CommunityToolkit.Mvvm.ComponentModel;
-using CommunityToolkit.Mvvm.Input;
-
 namespace SimpleCalculator;
 
 public partial class MainViewModel : ObservableObject
 {
-	private readonly IThemeService _themeService;
+    private readonly IThemeService _themeService;
```
Comment (nickrandolph) [inline reference/SimpleCalc/MVVM-CSharp/SimpleCalculator/MainViewModel.cs:5]: Can you make sure that spacing/tabs is consistent with rest of the code

Comment (pictos) [reply reference/SimpleCalc/MVVM-CSharp/SimpleCalculator/MainViewModel.cs]: @nickrandolph used tabs. But the `editorconfig` used by the template is set to space and not tabs. I found that's the default on our template.
image

Comment (nickrandolph) [reply reference/SimpleCalc/MVVM-CSharp/SimpleCalculator/MainViewModel.cs]: happy for the code to change to tabs but we should apply that across all code, not just on one file

**Comment by pictos** [comment]
@nickrandolph I believe it's good to go now

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**